src/test/java/ebay/carina/SearchTest.java

```diff
@@ -17,101 +16,134 @@
 
 public class SearchTest implements IAbstractTest {
     private static final Logger LOGGER = LoggerFactory.getLogger(SearchTest.class);
-    private final static String CATEGORIES_TITLE = "All Categories";
+
+    private static final String SIGNIN_PAGE_MSG = "Sign in to eBay or create an account";
     private final static String PRODUCTS_NOT_FOUND_MSG = "No exact matches found";
 
-    @Test()
+    @Test
     public void searchTest() {
         WebDriver driver = getDriver();
```

Comment:
You can put just getDriver() to constructors of pages, Carina will check that Default driver is already existed and shouldn't create new driver in this case